README.md

```diff
@@ -14,6 +14,7 @@ You can set the following environment variables for the dynamic ansible inventor
 export CHEF_USER=john
 export CHEF_PEMFILE=/home/john/.chef/john.pem
 export CHEF_SERVER_URL="https://my-chef-api"
+export CHEF_SERVER_SSL_VERIFY=false
```

Comment:
Could you change the value in the README to `true`? Just to avoid inattentive people from copy-pasting this and having an insecure default.

Reply:
Indeed it would be better :)
I just updated the PR